### ISSUE TITLE
`TypeError: network_test() missing 1 required positional argument: 'case'` when adding an `example`

### ISSUE THREAD

**Issue by pjrt** [kiwicom/schemathesis#386]
Hey. Not sure why this happens but when use the swagger file below, I get the following error:

```
TypeError: network_test() missing 1 required positional argument: 'case'
```

Removing the `example` field under `content.application/json` fixes it.  

```yaml
paths:
  /v1/some/request:
    post:
      summary: Summary
      requestBody:
        required: true
        content:
          application/json:
            schema:
              type: object
              required:
                - break
                - pod
                - id
              properties:
                break:
                  type: integer
                pod:
                  type: integer
                id:
                  type: string
                  format: uuid
              example:
                breakdur: 90
                podid: 4
                asstid: 9da17b9f-b029-4bfe-9640-d76009f6e717
      responses:
        "200":
          description: Request accepted
        "400":
          description: Invalid JSON
        "404":
          description: not found
        "5XX":
          description: Internal server error

```

**Comment by pjrt** [comment]
I figured it out. `example` is one level too low (it should be under `application/json` but I had it under `schema`). 

The error the cli gives isn't clear though. Closing.

**Comment by Stranger6667** [comment]
Hm, interesting - it should give an error from jsonschema validation. I'll check it, maybe we can improve the error message

**Comment by Stranger6667** [comment]
There are two bugs under the hood, unfortunately.

Since "schema" value is Schema object it can have "example" field (this is why built-in validation didn't complain) - https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.2.md#fixed-fields-20 and it should not fail as it does at the moment.

Another problem is that for "requestBody" examples are ignored (https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.2.md#fixed-fields-12)
Thank you very much @pjrt for reporting it, I will work on fixes

**Comment by Stranger6667** [comment]
And another issue is that the CLI output is not helpful at all - I will create a separate issue for it

**Comment by Stranger6667** [comment]
The original bug is solved in #393 
Better examples support is implemented in #395 
Expanding tracebacks option is done in #396

Changes will be released later today / tomorrow

Let us know if there is anything we can improve :)
